deployment-strategies/quarkus/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>deployment-strategies-quarkus</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Quarkus OpenShift TS: Deployment Strategies: Quarkus</name>
+
+    <properties>
+        <!-- this could also be set in application.properties, but this is much closer to what users actually do -->
+        <quarkus.kubernetes.deploy>true</quarkus.kubernetes.deploy>
+        <quarkus.kubernetes-client.trust-certs>true</quarkus.kubernetes-client.trust-certs>
```

Comment:
Why isn't this in `application.properties`?

Reply:
Because it's only meant for build time. The application doesn't use the Kubernetes client, it's only used during build when the Quarkus Kubernetes extension talks to the OpenShift cluster.

(Yes, I'm aware that the guide says this should be in `application.properties`. It's not wrong, but somewhat misleading IMHO.)

Reply:
Technically, both these properties could be in `application.properties`, and it would all work. That's actually what I had first :-)